pkg/controller/user-cluster-controller-manager/resources/reconciler.go

```diff
@@ -1054,8 +1054,10 @@ func (r *reconciler) reconcileDeployments(ctx context.Context, data reconcileDat
 	}
 
 	if r.isKonnectivityEnabled {
+		kResourcesOverrides := resources.GetOverrides(data.cluster.Spec.ComponentsOverride)
```

Comment:
nit: would we prefer having the names as konnectivityResources instead of kResources?

Reply:
sounds good to me! i renamed the variable in 13eba87cf01e6e2777dea21137e2e391b9025b78